custom_connectors/custom_auth/trackvia_connector.rb

```diff
@@ -0,0 +1,769 @@
+          control_type: 'select',
+          pick_list: 'views',
+          pick_list_params: { app_name: 'app_name' },
+          optional: false,
+          hint: 'Select an available view from the list above.'
+        }
+      ],
+
+      webhook_notification: lambda do |_input, payload|
+        hash = payload[0]
+        hash['ID'] = hash.delete('id')
+        hash
+      end,
+      webhook_subscribe: lambda do |webhook_url, _connection, input, _recipe_id|
+        post("/openapi/zapier/views/#{input['view_id']}/api/hooks",
+             target_url: webhook_url,
+             event: 'created')
+      end,
+
+      webhook_unsubscribe: lambda do |webhook, input|
+        delete("/openapi/zapier/views/#{input['view_id']}" \
+          "/api/hooks/#{webhook['id']}")
+      end,
+
+      output_fields: ->(object_definitions) { object_definitions['hook_body'] },
+
+      sample_output: lambda { |_connection, input|
+        call(:get_fields_sample_output, view_id: input['view_id'])
+      }
+    },
+
+    updated_record: {
+      description: "Updated <span class='provider'>" \
+      'record</span> in ' \
+      "<span class='provider'>TrackVia</span> view",
+      help: 'Triggers whenever a record belonging to ' \
+      'a specified TrackVia view is updated.',
+      type: :paging_desc,
+      config_fields: [
+        {
+          name: 'app_name',
+          label: 'App',
+          type: 'string',
+          control_type: 'select',
+          pick_list: 'apps',
+          optional: false,
+          change_on_blur: true,
+          hint: 'Select a TrackVia application from the list above'
+        },
+        {
+          name: 'view_id',
+          label: 'View',
+          type: 'integer',
+          control_type: 'select',
+          pick_list: 'views',
+          pick_list_params: { app_name: 'app_name' },
+          optional: false,
+          hint: 'Select an available view from the list above.'
+        }
+      ],
+
+      webhook_notification: lambda do |_input, payload|
+        hash = payload[0]
+        hash['ID'] = hash.delete('id')
+        hash
+      end,
+
+      webhook_subscribe: lambda do |webhook_url, _connection, input, _recipe_id|
+        post("/openapi/zapier/views/#{input['view_id']}" \
+        '/api/hooks',
+             target_url: webhook_url,
+             event: 'updated')
+      end,
+
+      webhook_unsubscribe: lambda do |webhook, input|
+        delete("/openapi/zapier/views/#{input['view_id']}" \
+          "/api/hooks/#{webhook['id']}")
+      end,
+
+      output_fields: ->(object_definitions) { object_definitions['hook_body'] },
+
+      sample_output: lambda { |_connection, input|
+        call(:get_fields_sample_output, view_id: input['view_id'])
+      }
+    },
+    deleted_record: {
+      description: "Deleted <span class='provider'>" \
+      'record</span> from view in ' \
+      "<span class='provider'>TrackVia</span>.",
+      help: 'Triggers whenever a record belonging to ' \
```

Comment:
Can this help?
`Trigger when record is deleted from TrackVia view`

Reply:
Talked with product team to fix labeling, please see changed code.